tests/SprykerTest/Zed/RabbitMq/Business/RabbitMqFacadeTest.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerTest\Zed\RabbitMq\Business;
+
+use Codeception\Test\Unit;
+use Spryker\Zed\RabbitMq\Business\Model\Queue\QueueInfo;
+use Spryker\Zed\RabbitMq\Business\RabbitMqBusinessFactory;
+use Spryker\Zed\RabbitMq\Business\RabbitMqFacade;
+
+/**
+ * Auto-generated group annotations
+ *
+ * @group SprykerTest
+ * @group Zed
+ * @group RabbitMq
+ * @group Business
+ * @group Facade
+ * @group RabbitMqFacadeTest
+ * Add your own group annotations below this line
+ */
+class RabbitMqFacadeTest extends Unit
+{
+    /**
+     * @var \Spryker\Zed\RabbitMq\Business\RabbitMqFacade
+     */
+    protected $rabbitMqFacade;
+
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|\Spryker\Zed\RabbitMq\Business\RabbitMqBusinessFactory
+     */
+    protected $rabbitMqBusinessFactoryMock;
+
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject|\Spryker\Zed\RabbitMq\Business\Queue\QueueInfo
+     */
+    protected $queueInfoMock;
+
+    /**
+     * @return void
+     */
+    protected function _before()
+    {
+        $this->rabbitMqBusinessFactoryMock = $this->createMock(RabbitMqBusinessFactory::class);
+        $this->queueInfoMock = $this->createMock(QueueInfo::class);
+
+        $this->rabbitMqFacade = new RabbitMqFacade();
+        $this->rabbitMqFacade->setFactory($this->rabbitMqBusinessFactoryMock);
```

Comment:
Please create them where they are used, since object are changed by reference this approach leads to errors (when several tests modify those objects) that are hard to catch.